ccd-definition/AuthorisationCaseField/CareSupervision/barrister.json

```diff
@@ -828,6 +828,13 @@
     "UserRole": "[BARRISTER]",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "isC2Confidential",
+    "UserRole": "[BARRISTER]",
```

Comment:
should include `caseworker-publiclaw-systemupdate` as well?

Reply:
how about gatekeeper, judiciary and magistrate?

Reply:
oh , it is a temporary field. So we don't need it right?

Reply:
yeah just temporary field, think there is no need to include systemupdate. and gatekeeper, judiciary and magistrate can't access this event.